graphql-parser/build.rs

```diff
@@ -17,25 +18,29 @@ use graphql_parser::parse_query;
 use graphql_parser::parse_schema;
 
 \n\n");
-
-  for ent in read_dir("tests")? {
-    let entry = ent?;
-    let mut path = entry.path();
-    if let (Some(osname), Some(ext)) = (path.file_stem(), path.extension()) {
-      let name = osname.to_str().unwrap().to_owned();
-      if name.contains(".") || ext != "graphql" { continue }
-      let src = entry.file_name().to_str().unwrap().to_owned();
-      let canonical_src = format!("{}.canonical.graphql", name);
-      path.pop();
-      path.push(canonical_src.clone());
-      output.push_str(
-        &format!(
-          "test!({}, include_str!(\"{}\"), include_str!(\"{}\"));\n",
-          &name, &src, if path.is_file() { &canonical_src } else { &src }
-        )
-      )      
+    let mut lines: Vec<String> = Vec::new();
```

Comment:
Can we call this "tests" or something? I'd like to avoid a footgun where we try to push two lines and everything goes to hell.

Alternately, sort the dirents before looping over them

Reply:
Sure thing!

Reply:
went with the rename, sorting the dirents is quite a PITA because of all the Result wrapping and getting down to the name.